roadmap/views.py

```diff
@@ -1,3 +1,22 @@
-from django.shortcuts import render
+from rest_framework.response import Response
+from rest_framework.views import APIView
 
-# Create your views here.
+from roadmap.models import LocalDomain, MetaDomain
+from roadmap.serializers import LocalDomainSerializer, MetaDomainSerializer
+
+
+class DomainListView(APIView):
```

Comment:
Let's use `ListAPIView` generic (move all logic from _get_ to `list`).
Also use next pattern: {view_name}{view_class} (for instance, `DomainListAPIView`) for the view name.

Reply:
Good point. Done